### PR TITLE
feat(npm-audit-fix): Add audit report to created pull request

### LINKED ISSUE #343: Write npm audit fix into PR description
If I see a PR like https://github.com/nextcloud/mail/pull/9569 it's always extra work to find out what changed. I propose to write the output of the npm audit fix command into the description of the PR.

@nickvergessen objections?

### PR DESCRIPTION
**Pull request by susnux**
* Resolves #343 

## An example looks like this
image

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
🐘

File: workflow-templates/npm-audit-fix.yml [hunk 1]
```diff
@@ -47,8 +47,8 @@ jobs:
         run: npm i -g npm@"${{ steps.versions.outputs.npmVersion }}"
 
       - name: Fix npm audit
-        run: |
-          npm audit fix
+        id: npm-audit
+        uses: susnux/npm-audit-action@2a60bd2e79cc77f2cc4d9a3fe40f1a69896f3a87 # v0.1.0
```
Comment (ChristophWurst) [inline workflow-templates/npm-audit-fix.yml:51]: if this is used in nextcloud repos I would prefer moving https://github.com/susnux/npm-audit-action to the commong organization too. Else we run into the problem again where we find an issue with an action but have no permission to send a fix.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: workflow-templates/npm-audit-fix.yml (hunk 1 above)
Comment (susnux) [reply workflow-templates/npm-audit-fix.yml:51]: Sure just created under my handle just like with skjnldsv permissions action.
So under which organization would you prefer it?

Comment (ChristophWurst) [reply workflow-templates/npm-audit-fix.yml:51]: fair point. we should move them all, also skjnldsv actions :wink: 

we can use https://github.com/nextcloud or https://github.com/nextcloud-libraries/ if we want more control over who can push

Comment (susnux) [reply workflow-templates/npm-audit-fix.yml:51]: cc @skjnldsv

Comment (skjnldsv) [reply workflow-templates/npm-audit-fix.yml:51]: > also skjnldsv actions

except I am publishing them on the github store and they are not related to the NC workload :)

Comment (susnux) [reply workflow-templates/npm-audit-fix.yml:51]: So what's the conclusion? Should I move it to nextcloud or nextcloud-libraries? :)

Comment (susnux) [reply workflow-templates/npm-audit-fix.yml]: @ChristophWurst I agree, moved it.

**Pull request merged**